### ISSUE TITLE
commit history 404

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wcag#1785]
When looking for changes, I noticed a couple of things. Maybe just due to draft state &/or staging?

* [Changelog](https://w3c.github.io/wcag/guidelines/22/#changelog) missing recent changes.

* Link not working in:
> The full [commit history to WCAG 2.2](https://github.com/w3c/wcag/commits/master/guidelines) is available.

**Comment by kazuhito-kidachi** [comment]
I found that Commit history link refers old URL, which contains invalid branch name of "master", too.

**Comment by shawna-slh** [comment]
Hi @michael-n-cooper Would you fix this with the next publication?

In the index page for understanding and techniques, change master to main:
> ```<p>For a more detailed view of recent changes to the informative documents see the <a href="https://github.com/w3c/wcag/commits/master">github updates</a>.</p>```

change to:
> ```<p>For a more detailed view of recent changes to the informative documents see the <a href="https://github.com/w3c/wcag/commits/main">github updates</a>.</p>```

**Comment by iadawn** [comment]
This is all fixed
